programs/survival/what_biome_on_block.sc

```diff
@@ -0,0 +1,79 @@
+// shows biomes blockwise. useful for marking out rivers, beaches, soul sand valley and stuff.
+// now works across diemsions (yes even in the_end)
+__config()->{
+    'stay_loaded'->true,
+    'scope' -> 'global',
+    'commands' -> {
+        '' -> _() -> print(format('r Missing arguments. Correct syntax /showbiomes <toggle>')),
+        '<toggle>' -> l('__toggle_markings'),

```

Comment:
You don't need to input a list if you are not also inputting default values for missing arguments. Also, you can use `[·]` intead of `l(·)`.

Reply:
gotcha. I once read that scarpet has to convert all the [] into l() and {} in to m(). is that not the case anymore ?

Reply:
It does, at compile time. There's no reason to remove readability by doing it manually.